<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>30</y>
      <width>471</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="drive_verticalLayout"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>609</x>
      <y>10</y>
      <width>481</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGraphicsView" name="graphicsView"/>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="font">
        <font>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Drive</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>SQ Activity</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Performance</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>120</width>
      <height>731</height>
     </rect>
    </property>
    <property name="title">
     <string>CPU Selector</string>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>111</width>
       <height>701</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="cpu_formLayout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Operation">
    <property name="title">
     <string>&amp;Operation</string>
    </property>
    <addaction name="action_Init"/>
    <addaction name="action_Start"/>
    <addaction name="actionS_top"/>
    <addaction name="actionE_xit"/>
   </widget>
   <addaction name="menu_Operation"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Init">
   <property name="text">
    <string>&amp;Init</string>
   </property>
  </action>
  <action name="action_Start">
   <property name="text">
    <string>&amp;Start</string>
   </property>
  </action>
  <action name="actionS_top">
   <property name="text">
    <string>S&amp;top</string>
   </property>
  </action>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
